src/test/tv/codely/scala_http_api/module/video/infrastructure/repository/InMemoryVideoRepositoryTest.scala

```diff
@@ -0,0 +1,19 @@
+package tv.codely.scala_http_api.module.video.infrastructure.repository
+
+import tv.codely.scala_http_api.module.video.VideoIntegrationTestCase
+import tv.codely.scala_http_api.module.video.domain.VideoStub
+
+final class InMemoryVideoRepositoryTest extends VideoIntegrationTestCase {
+  "In memory video repository" should {
+    "search all existing videos" in {
+      val video        = VideoStub.random
+      val anotherVideo = VideoStub.random
+      val videos       = Seq(video, anotherVideo)
+
+      repository.save(video)
+      repository.save(anotherVideo)
+
+      repository.all() should be(videos)
```

Comment:
We should use always the same matcher style. Until now we've being using the `repository.all() shouldBe videos` style. If you prefer this `repository.all() should be(videos)` alternative, we should be consistent in all our code base and change for this one.

What do you prefer?

Reply:
We didn't use `shouldBe` at all :P

Reply:
😕 🤔 

![monosnap 2017-12-08 11-56-47](https://user-images.githubusercontent.com/986235/33762952-e459ce88-dc0e-11e7-88e1-1fe2750dbb14.png)

Reply:
Ahhh, true! xD